README.md

```diff
@@ -10,12 +10,12 @@ Symbol CLI tool that allows you creating, configuring and running Symbol&#39;s c
 [![Coverage Status](https://coveralls.io/repos/github/nemtech/symbol-bootstrap/badge.svg?branch=main)](https://coveralls.io/github/nemtech/symbol-bootstrap?branch=main)
 [![Api Doc](https://img.shields.io/badge/api-doc-blue.svg)](https://nemtech.github.io/symbol-bootstrap/)
 
-
 <!-- toc -->
 * [symbol-bootstrap](#symbol-bootstrap)
 * [Why this tool?](#why-this-tool)
 * [Key benefits:](#key-benefits)
 * [Concepts](#concepts)
+* [custom repeat preset for bootstrap preset.](#custom-repeat-preset-for-bootstrap-preset)
```

Comment:
```suggestion
```
If you link to this level-5 header you'll have to include links to all other level-5 headers :)

Reply:
Let's rework the custom prests examples on https://github.com/nemtech/symbol-bootstrap/issues/37